scripts/recreate_records

```diff
@@ -36,26 +36,6 @@ if [ ! -x "$(command -v celery)" ]; then
   exit 1
 fi
 
-CELERY_LOGLEVEL=$1
-if [ -z $1 ]; then
-	CELERY_LOGLEVEL="ERROR"
-fi
-
-cleanup() {
-	echo "----> Killing Celery Worker ${CELERY_PID}"
-	kill "${CELERY_PID}"
-}
-
-CELERY_PIDFILE="${VIRTUAL_ENV}/worker.pid"
-if [ -f "${CELERY_PIDFILE}" ]; then
-	echo "----> Purging all pending messages"
-	celery purge -f -A inspirehep.celery --workdir="${VIRTUAL_ENV}"
-else
-	celery worker -E -A inspirehep.celery --loglevel=${CELERY_LOGLEVEL} --workdir="${VIRTUAL_ENV}" --autoreload --pidfile="${CELERY_PIDFILE}" --purge &
-	CELERY_PID=$!
-	trap 'cleanup' 0 1 2 3 6
-fi
-
```

Comment:
As a general indication, but not something that should be done now, @jalavik suggests that we keep the doors open for the developers that don't want to ride the Docker train. So this script will have to be split in two cases, one run by docker, one run by non-Docker users.